Modules/_datetimemodule.c

```diff
@@ -2876,13 +2876,18 @@ date_fromtimestamp(PyObject *cls, PyObject *obj)
                                 cls);
 }
 
-/* Return new date from current time.
- * We say this is equivalent to fromtimestamp(time.time()), and the
- * only way to be sure of that is to *call* time.time().  That's not
- * generally the same as calling C's time.
- */
+/*[clinic input]
+@classmethod
+datetime.date.today
+
+Return new date from current time.
```

Comment:
This can also be a datetime, since datetime subclasses date.

This PR otherwise retains the existing doc-strings, which I suggest doing consistently.